access_group.go

```diff
@@ -153,6 +153,15 @@ type AccessGroupSAML struct {
 	} `json:"saml"`
 }
 
+// AccessGroupAzureAuthContext is used to configure access based on Azure auth contexts.
+type AccessGroupAzureAuthContext struct {
+	AuthContext struct {
```

Comment:
can we use external structs here? inline structs are hard to use inline.

Reply:
For code consistency I'd say not in this PR if possible. We currently define all our groups this way

Reply:
ah you're right. we can look at that another time.